src/views/home/home.view.jsx

```diff
@@ -162,6 +162,9 @@ function Home ({
     <div className={classes.root}>
       <Container backgroundColor={theme.palette.primary.main} addHeaderPadding disableTopGutter>
         <section className={classes.section}>
+          {ethereumNetworkTask.data.chainId === 4 && (
```

Comment:
Why only when chainId === 4?

Reply:
Because in production we only support Rinkeby and Mainnet and Chami wants to display the label in Rinkeby but not in Mainnet (as Uniswap does).